src/db/seeders/trackgroup-seeder.js

```diff
@@ -13,6 +13,14 @@ const generateTracks = async (trackgroup) => {
         status: 'free',
         date: faker.date.past(1)
       })
+      await Promise.all(Array(faker.datatype.number(1, 5)).fill().map(async (v, i) => {
```

Comment:
Radomly create between 1 and 5 plays per song for the user with `displayName` listener